packages/sanity/src/core/config/types.ts

```diff
@@ -408,32 +413,36 @@ export interface SourceOptions extends PluginOptions {
   apiHost?: string
 
   /** @internal */
-  auth?: AuthStore
+  auth?: AuthConfig | AuthStore
```

Comment:
Since we're mentioning this in release notes - should this now be marked as `@beta`?

Reply:
I guess ideally we'd only want to mark `AuthStore` as internal

Reply:
True, thanks. Will fix.